db/yoyo.ini

```diff
@@ -0,0 +1,9 @@
+# List of migration source directories. "%(here)s" is expanded to the
```

Comment:
It doesn't look like this file is used when you're using it in python mode. This is just here so we can create a new migration or run "yoyo" on the CLI? I'm not sure. I'm not sure if that's something we'd ever do with this project though.

Reply:
Yeah, looks like you are correct. Deleting it doesn't change anything. I thought at first the `verbosity` setting was doing something, but I think with the Python mode, we just use `LOG_LEVEL`. I'll remove this for now, I think we can just generate new migration files manually and test.